Add credit for integration test example (#434)
NixOS/nix.dev: source/tutorials/integration-testing-using-virtual-machines.md

- A VM named `client` running HTTP client queries using `curl`.
- A `testScript` orchestrating testing logic between `client` and `server`.

> … using an example Nix expression adapted from [How to use NixOS for lightweight integration tests](https://www.haskellforall.com/2020/11/how-to-use-nixos-for-lightweight.html).

Reviewer: ```suggestion
The Nix expression example is adapted from [How to use NixOS for lightweight integration tests](https://www.haskellforall.com/2020/11/how-to-use-nixos-for-lightweight.html).
```
